test/clj/puppetlabs/trapperkeeper/services/webserver/jetty9_core_test.clj

```diff
@@ -50,6 +50,15 @@
         (format "Expected gzipped response, got this response: %s"
                 resp))))
 
+(defn validate-gzip-encoding-when-gzip-requested-on-post-requests
+  [body port]
+  ;; The client/get function asks for compression by default
```

Comment:
Is this comment still valid?

Reply:
Based on this note in the docs for clj-http-client:
```
:decompress-body: optional; if true, an 'accept-encoding' header with a value of 'gzip, deflate' will be added to the request, and the response will be automatically decompressed if it contains a recognized 'content-encoding' header. Defaults to true.
```
Sounds like it applies to all requests.